R/pkg/R/mllib.R

```diff
@@ -602,14 +599,14 @@ setMethod("spark.survreg", signature(data = "SparkDataFrame", formula = "formula
 # Returns a summary of the AFT survival regression model produced by spark.survreg,
 # similarly to R's summary().
 
-#' @param object A fitted AFT survival regression model
+#' @param object a fitted AFT survival regression model.
 #' @return \code{summary} returns a list containing the model's coefficients,
 #' intercept and log(scale)
 #' @rdname spark.survreg
 #' @export
 #' @note summary(AFTSurvivalRegressionModel) since 2.0.0
 setMethod("summary", signature(object = "AFTSurvivalRegressionModel"),
-          function(object, ...) {
+          function(object) {
```

Comment:
I guess you meant omitting `...` in the generic function as well? If so, the S3 base `summary` would not be added as a default method when it sees S4 generic definition and would so be masked by SparkR.